internal/cache/metadata/stat_cache_test.go

```diff
@@ -408,3 +418,128 @@ func (t *MultiBucketStatCacheTest) Test_ExpiresLeastRecentlyUsed() {
 	assert.Equal(t.T(), cardamom, spices.LookUpOrNil("cardamom", someTime))
 	assert.Equal(t.T(), saffron, spices.LookUpOrNil("saffron", someTime))
 }
+
+func (t *StatCacheTest) Test_Create_Entry_When_No_Entry_Is_Present() {
```

Comment:
naming of test methods

Test_methodNameUnderTest_conditionAndExpectedResult or
methodNameUnderTest_condition_expectedResult

See what is followed in gcsfuse and name appropriately.

Reply:
added Test_methodNameUnderTest_ as per convention